nnf/__init__.py

```diff
@@ -998,6 +998,53 @@ def pair(node: NNF) -> NNF:
 
         return pair(sentence)
 
+    def project(self: T_NNF, vars: 't.FrozenSet[Name]') -> 'NNF':
```

Comment:
I've been avoiding `vars` as a variable name because it shadows the `vars()` builtin. I'm not sure if it really matters, but maybe `names` would be better?